doc/examples/bundle_assignment_maps.py

```diff
@@ -0,0 +1,111 @@
+"""
+====================================
+BUAN Bundle Assignment Maps Creation
+====================================
+
+This example explains how we can use BUAN [Chandio2020]_ to create assignment
+maps on a bundle. Divide bundle into N smaller segments.
+
+
+First import the necessary modules.
+"""
+
+from dipy.data.fetcher import get_two_hcp842_bundles
```

Comment:
from `dipy.data` instead of `from dipy.data.fetcher`

Same comment below

Reply:
@skoudoro 

`from dipy.data import get_two_hcp842_bundles` gives following error

`ImportError: cannot import name 'get_two_hcp842_bundles' from 'dipy.data' (/Users/../dipy/dipy/data/__init__.py)`

Reply:
it means we forgot to add it in `dipy/data/__init__.py` with the others. Could you add it?

Reply:
Added, it works now, thanks.